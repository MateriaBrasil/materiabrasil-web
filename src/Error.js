import React from 'react'
import Typography from '@material-ui/core/Typography'

export default ({ style, children, ...props }) =>
  children ? (
    <div style={{ ...style, padding: 20, textAlign: 'center' }}>
      <Typography color="error" variant="headline" {...props}>
        {children}
      </Typography>
    </div>
  ) : null
