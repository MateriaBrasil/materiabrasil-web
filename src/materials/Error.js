import React from 'react'

import Info from './Info'

export default ({ style, ...props }) => (
  <Info style={{ ...style }} color="error" {...props} />
)
