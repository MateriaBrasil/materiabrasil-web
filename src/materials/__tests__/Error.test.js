import React from 'react'
import renderer from 'react-test-renderer'

import Error from '../Error'

it('renders correctly', () => {
  const tree = renderer.create(<Error>Error</Error>).toJSON()
  expect(tree).toMatchSnapshot()
})
