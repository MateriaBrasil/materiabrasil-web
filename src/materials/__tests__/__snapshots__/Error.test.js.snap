// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "padding": 20,
      "textAlign": "center",
    }
  }
>
  <h1
    className="MuiTypography-root-1 MuiTypography-headline-6 MuiTypography-colorError-24"
  >
    Error
  </h1>
</div>
`;
